transforms/images/rolling-ball-plugin/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "polus-transforms-images-rolling-ball"
+version = "1.1.0-dev0"
+description = "A WIPP plugin to perform background subtraction using the rolling-ball algorithm."
+authors = ["Najib Ishaq <najib.ishaq@axleinfo.com>"]
```

Comment:
Add yourself as an author here, and change my email to use the `nih.gov` domain.